apex_launchtest/apex_launchtest/__init__.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 Apex.AI, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from .apex_launchtest_main import apex_launchtest_main
+
+from .apex_runner import ApexRunner
+from .decorator import post_shutdown_test
+from .io_handler import ActiveIoHandler, IoHandler
+from .proc_info_handler import ActiveProcInfoHandler, ProcInfoHandler
+from .ready_aggregator import ReadyAggregator
```

Comment:
so there is still lots of Apex names in the code constructs? Do we wanna keep these?

Reply:
ApexRunner is the biggest offender, and I actually don't expect anybody to use this class outside of the apex_rostest implementation (it's not for writing tests)  I'll just take it out of the __init__.py file.

Reply:
@dejanpan done - made apex_rostest_main and ApexRunner 'private' by taking them out of the __init__.py